poetry.lock

```diff
@@ -198,17 +194,17 @@ python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*"
 
 [[package]]
 name = "flake8"
-version = "3.9.2"
+version = "4.0.1"
```

Comment:
IIRC flake8 4.x doesn't coexist nicely with newer versions of Sphinx. Was up-reving this intentional? Is this why Sphinx got downgraded from 4.5.0 to 4.3.2 below?